### DISCUSSION TITLE
First generation GSync Support

### DISCUSSION THREAD

**Discussion by noaccOS** [NVIDIA/open-gpu-kernel-modules#386] [Q&A] [answered]
I saw it was mentioned that GSync support is planned but not yet available.

I own a first generation GSync monitor and was wondering if the open driver will only support the new standard or if my monitor will also be supported

**Comment by aritger** [comment, marked as answer]
We expect to be able to provide the same level of GSync support with the open kernel drivers as the closed kernel drivers, once we finally have GSync working with the open kernel drivers (and I unfortunately don't yet have an ETA).
